docs/install.rst

```diff
@@ -45,8 +41,20 @@ values in an environment variable of the same name.
 Tests
 =====
 
-To run tests:
+Tests depend on `hatch` for package management. To install hatch:
+
+.. code:: bash
+
+    $ pipx install hatch
```

Comment:
I wonder if it's worth putting hatch into dev reqs? No strong opinion on this tho